### DISCUSSION TITLE
Termux: Flyctl Error

### DISCUSSION THREAD

**Discussion by ghost** [TheCaduceus/CloneBot_V2#54] [Q&A] [answered]
Package flyctl not found in termux. Any other method bro?

**Comment by TheCaduceus** [comment, marked as answer]
**Some devices are not able to install `flyctl` because it does not support devices which are based on `i686` & `arm` because they are blacklisted [here](https://github.com/termux/termux-packages/blob/754151dd27b0df044e26d283b66dd37c9eacea95/packages/flyctl/build.sh#L10)**
